netjsonconfig/backends/openwrt/converters/wireless.py

```diff
@@ -114,13 +114,6 @@ def __intermediate_encryption(self, wireless):
             uci['key'] = encryption['key']
         # add ciphers
         cipher = encryption.get('cipher')
-        if (
-            protocol == 'wpa3_personal'
-            or protocol == 'wpa3_enterprise'
-            or protocol == 'wpa2_personal_mixed'
-            or protocol == 'wpa2_enterprise_mixed'
-        ):
-            cipher = 'ccmp'
```

Comment:
Initially, I asked @masap to add this. But after our discussion around MFP, I concluded updating the schema would be better.